addons/captives/stringtable.xml

```diff
@@ -247,6 +247,7 @@
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_name">
             <English>Require surrendering</English>
+            <German>Benötigt Kapitulation</German>
```

Comment:
In diesem Kontext sollte es bei "Kapitulation" bleiben. Ich habe Arma 3 nun einige Zeit nichtmehr angerührt, allerdings bin ich der Meinung, dass hier ausgedrückt werden soll:
Bevor jemand gefangen genommen werden kann, muss dieser erst kapituliert haben. Dementsprechend: "Benötigt Kapitulation"
Bitte um Korrektur im Falle der Irrens.

@bux578 Diesmal mach ich die anderen 2 PR am Wochenende fertig; und diesesmal wirklich ;)